packages/zosfiles/src/api/methods/mount/doc/IMountFsOptions.ts

```diff
@@ -0,0 +1,30 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Interface for the options to the mount file-system API.
+ * @export
+ * @interface IMountFsOptions
+ */
+export interface IMountFsOptions {
+
+    /**
+     * The file system type to mount
+     * @type {string}
+     */
+    "fs-type"?: string;
+
+    /**
+     * The mode for mounting the file system
+     * @type {string}
+     */
+    mode?: string;
```

Comment:
Since you have restricted this value to the supported values in the doc... how about we create a type for it? Helps out the programmer.

Reply:
Added type declaration in IMountFsMode.ts:
```typescript
export type IMountFsMode =
    "rdonly" |
    "rdwr";
```